torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -653,6 +655,15 @@ RegisterOperators reg(
            handler(ss.str());
          },
          aliasAnalysisSpecialCase()),
+     OperatorGenerator(
+         TORCH_SELECTIVE_SCHEMA(
+             "prim::ModuleDictIndex(Any self, str ind) -> Any"),
```

Comment:
Could you add a comment to this to the extent that this op is generated in the compiler, and self and the output are expected to be set to a ModuleType/ModuleTypeInterface ? I think if you look up TupleIndex there is a similar comment to that extent